docs/specifications/tests/Address-TP/address01.md

```diff
@@ -30,59 +30,10 @@ The domain name to be tested.
 2. Each IP address has to be checked against the blocks listed in the three
    tables below.
  
```

Comment:
The tables are not there anymore in the updated version. Please update, @sandoche2k.